notebooks/community/structured_data/rapid_prototyping_bqml_automl.ipynb

```diff
@@ -0,0 +1,1527 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #7.    curl https://archive.ics.uci.edu/ml/machine-learning-databases/abalone/abalone.data --output ${FILE_NAME}
</pre><p> </p><p>These should live in gs://cloud-samples-data/vertex-ai/ somewhere. Can you put it there if it already doesn't exist (do a search for "abalone")</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/80/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>